vllm_ascend/quantization/quant_config.py

```diff
@@ -89,6 +90,11 @@ def override_quantization_method(cls, hf_quant_cfg,
 
     def get_quant_method(self, layer: torch.nn.Module,
                          prefix: str) -> Optional["QuantizeMethodBase"]:
+        vllm_config = get_current_vllm_config()
+        model_type = vllm_config.model_config.hf_config.model_type
+        if model_type in packed_modules_model_mapping:
```

Comment:
```suggestion
        if model_type in packed_modules_model_mapping.keys():
```
maybe should be this ?